fix_simulation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use Rocky Linux 9.3 as the base image and set platform to linux/amd64
+FROM --platform=linux/amd64 rockylinux:9.3
```

Comment:
Does it need to be amd64? I thought python could run on arm too.

Reply:
Yeah but quickfix doesnt run on arm64. Added platform amd64 config for docker to assume that environment rather than default to that of the machine running the container.